### ISSUE TITLE
Extra systemd mounts prevent systems rebooting after #4455

### ISSUE THREAD

**Issue by MilhouseVH** [OpenELEC/OpenELEC.tv#4483]
After #4455, if you add a systemd mount and try to reboot, the system will hang while rebooting.

Add an nfs mount (following is an example, change paths etc. to suit circumstances):

```
[Unit]
Description=test nfs mount script
Requires=network-online.service
After=network-online.service
Before=kodi.service

[Mount]
What=192.168.1.66:/mnt/sdb2/PVR
Where=/storage/pvr
Options=
Type=nfs

[Install]
WantedBy=multi-user.target
```

Once the mount service has started and mounted the NFS share, try to reboot the system - It will stop all the services, then hang before rebooting.

If the connman-1.30 bump is reverted, the system does reboot normally. If there is no systemd mount, the system will also reboot normally with connman-1.30.

However, with connman-1.30 AND systemd mount, the system will hang at reboot.

**Comment by mlade77** [comment]
Btw. in my case that happens only when I am using WiFi dongle. 
If I use cable that doesn't happen.
https://github.com/OpenELEC/OpenELEC.tv/issues/4504

**Comment by the-dreamer** [comment]
in my case i have this problems only when i have more than one mount too use. my current workaround seems to be adding some mounts directly in kodi. my pvr / recordings folder is mounted on startup. the other mounts (music & pictures) directly in kodi without mount script. 

reboot/shutdown hangs seems to be gone. tried with 01.01.2016 milhouse build and OE7 fritsch egl build. 
btw. i saw in my scripts leftovers from xbmc which i have renamed now.

**Comment by MainUser1** [comment]
In my case this also happens when using a wired connection.

I think it should be noted, that network booting OpenELEC from NFS also breaks reboot / shutdown.
There doesn't seem to be a usable workaround in this situation.

I'm more than willing to help if there is something I can do to track this issue down.

**Comment by MilhouseVH** [comment]
If you have this problem can you test build #0205 as this build contains a potential fix, and report your results:

RPi1/RPi2: http://forum.kodi.tv/showthread.php?tid=250817
x86: http://forum.kodi.tv/showthread.php?tid=238393

**Comment by chewitt** [comment]
closing as resolved in #4659
